wger/manager/models.py

```diff
@@ -738,17 +738,15 @@ def get_owner_object(self):
         '''
         return self
 
-    def get_workout_session(self, date=None):
+    def get_workout_session(self):
         '''
         Returns the corresponding workout session
 
         :return the WorkoutSession object or None if nothing was found
         '''
-        if not date:
-            date = self.date
-
         try:
-            return WorkoutSession.objects.filter(user=self.user).get(date=date)
+            return WorkoutSession.objects.filter(
```

Comment:
@reifred I can see that you removed the explicit link of date between workout session and workout log and added the foreign key of the workout log as per the second suggestion in story description and as noted I can still create workouts. I believe this change will be of benefit towards database management activities.